x/ccv/provider/keeper/keeper.go

```diff
@@ -1264,8 +1267,10 @@ func (k Keeper) GetConsumerCommissionRate(
 	}
 
 	cr := sdk.Dec{}
+	// handle error gracefully since it's called in BeginBlockRD
 	if err := cr.Unmarshal(bz); err != nil {
-		panic(fmt.Sprintf("consumer commission rate unmarshalling failed: %s", err))
+		k.Logger(ctx).Error("consumer commission rate unmarshalling failed: %s", err)
```

Comment:
Would this failing to unmarshal mean we have corrupted state? If this is the case, maybe `panic` is better than moving forward.

Reply:
TBD when @mpoke is back I guess.